en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -46,6 +49,53 @@ https://youtu.be/aEzFKPMEfjc
 https://youtu.be/lAjjTFFZebI
 {% endyoutube %}
 
+
+
+## Multiple Vehicles with Gazebo based on RTPS {#with_rtps}
+
+To simulate multiple vehicles based on RTPS in Gazebo, use `gazebo_sitl_multiple_run.sh` command in the terminal as described above with `-t px4_sitl_rtps` option (from the root of the *Firmware* tree).  For RTPS, the `micrortps_agent` should be run in the different terminal  for each vehicle.
+
+```
+Tools/gazebo_sitl_multiple_run.sh  -t px4_sitl_rtps [-m <model>] [-n <number_of_vehicles>] [-w <world>] [-s <script>]
+```
+
+> **Note** The model "iris_rtps" is only implemented until now.
+
+
```

Comment:
Add a note stating that you need to have RTPS  installed, and link to the installation instructions.

Reply:
I will add it. (DONE)